### ISSUE TITLE
Bug - Button - Focus accessibility issue

### ISSUE THREAD

**Issue by Arathy-s** [patternfly/patternfly#6428]
**Describe the problem**
Focus on the button is difficult to perceive.

**How do you reproduce the problem?**
https://www.patternfly.org/components/button

**Expected behaviour**
There should be a clear visual focus indicator for all the interactive elements.

**Screenshots**
Before focus:
<img width="1440" alt="Screenshot 2024-03-18 at 7 04 00 PM" src="https://github.com/patternfly/patternfly-react/assets/49025521/49c7ebf8-622d-4871-a782-f08593c5d10b">

On Focus:
<img width="1440" alt="Screenshot 2024-03-18 at 7 04 31 PM" src="https://github.com/patternfly/patternfly-react/assets/49025521/5fc4ca6d-b2d4-4377-ae9a-ae0ba0e8a7cf">

**What is your environment?**
 - OS: Mac OS Sonoma 14.3
 - Browser: Chrome
 - Version: 122.0.6261.129 (Official Build) (x86_64)

**What is your product and what release date are you targeting?**

**Any other information?**

**Comment by srambach** [comment]
After discussion with design, expanding the focus ring for buttons out a bit seems the best solution.
